### PR TITLE
refactor(package.json): update package.json dev environment scripts

### LINKED ISSUE #132: Update package.json
The current `package.json` file has many npm commands that do not work, I guess they were from a previous version of Origami and do not work with the current directory structure anymore. We should either update the package.json file or we should remove these scripts entirely.

image

**I would like to work on this.**

### PR DESCRIPTION
**Pull request by fristonio**
[WIP]

- Move prettier and eslint-plugin-prettier to dev dependencies from dependencies.
- Fix npm scripts to run origami in dev environment. Make `npm start`, `npm run dev` work.
- Fix lint scripts in package.json
- Remove test scripts from package.json
- Fixes #132

### CODE REVIEW

**Comment by fristonio** [comment]
@AvaisP @uttu357 It would be great if you could give your suggestions :) I am now working on adding eslint configuration to Origami.

**Comment by fristonio** [comment]
I have also added eslint configuration `.eslintrc`. I think the PR is now complete. I have not resolved any eslint issues as of now, I think it would be best if we do that in another PR. Also if you see any issues with `eslintrc` please let me know.

**Comment by fristonio** [comment]
@uttu357 I have also added a dummy test script. Please have a look.

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    "react/no-is-mounted": 1,
+    "react/no-set-state": 0,
+    "react/no-unknown-property": 1,
+    "react/prefer-es6-class": 1,
+    "react/prop-types": 1,
+    "react/react-in-jsx-scope": 1,
+    "react/self-closing-comp": 1,
+    "react/sort-comp": 1
+  }
+}
```
Comment (uttu357) [inline .eslintrc]: Endline missing.

**Pull request merged**